feature/settings/src/main/kotlin/org/meshtastic/feature/settings/SettingsScreen.kt

```diff
@@ -327,7 +328,7 @@ fun SettingsScreen(
                         Intent(Intent.ACTION_CREATE_DOCUMENT).apply {
                             addCategory(Intent.CATEGORY_OPENABLE)
                             type = "application/csv"
-                            putExtra(Intent.EXTRA_TITLE, "Meshtastic_rangetest_$timestamp.csv")
+                            putExtra(Intent.EXTRA_TITLE, "Meshtastic_rangetest_$timestamp_$nodeName.csv")
```

Comment:
nodeName, then timestamp I think will sort better. 

And I think short name rather than long. Some people use 40 character longnames, that's about an 90 character filename, windows' total is 260 ...